src/utils/helpers.ts

```diff
@@ -0,0 +1,8 @@
+export const getEnv = (name: string) => {
```

Comment:
rather than the constants file above could we make this function take a string literal type with the expected env vars?  Something like:

`envVars: "ANALYSIS_BUCKET_NAME" | "AUDIT_BUCKET_NAME"
`

Reply:
Done, please review!